vendor/engines/secure_rooms/app/controllers/secure_rooms_api/events_controller.rb

```diff
@@ -2,11 +2,37 @@ module SecureRoomsApi
 
   class EventsController < SecureRoomsApi::ApiController
 
+    before_action :parse_time
+
     def create
-      Rails.logger.info "SecureRoomsApi Event: #{params}"
+      SecureRooms::Alert.create(alert_params)
```

Comment:
And we should probably do some sort of if/else. I think I'd want to know if this fails. At least a rollbar error, even if we return an OK to the client.